src/types/none.rs

```diff
@@ -10,8 +11,11 @@ pyobject_native_type_extract!(PyNone);
 impl PyNone {
     /// Returns the `None` object.
     #[inline]
-    pub fn get(py: Python<'_>) -> &PyNone {
-        unsafe { py.from_borrowed_ptr(ffi::Py_None()) }
+    pub fn get<'py>(py: Python<'py>) -> Borrowed<'py, 'py, PyNone> {
+        unsafe {
+            let bound = ffi::Py_None().assume_borrowed(py);
+            std::mem::transmute(bound)
```

Comment:
If you rebase on `main`, we literally just added `.downcast_unchecked()` method to `Borrowed`, which you can use to avoid the `transmute`.

Reply:
awesome